api/Node.json

```diff
@@ -135,10 +135,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "7"
+              "version_added": "≤4"
             },
             "safari_ios": {
-              "version_added": "7"
+              "version_added": "1"
```

Comment:
Is this based on testing iOS 1? Safari ≤4 + iOS 1 is an unusual pairing of versions.

Reply:
The generated range was actually "≤3" since that's the earliest version the collector will run in, and Safari 3 = iOS 1, but since we don't have a "≤3" range I had to bump it up to "≤4".

Reply:
I see, that explains all of the recent Safari ≤4 + iOS 1 changes.